src/firmware-update.html

```diff
@@ -0,0 +1,26 @@
+---
+layout: base
+description: Zigbee and Thread USB stick by the creators of Home Assistant
+---
+
+<div class="container">
+  <div>
+    <h1>Firmware update</h1>
+    <p>First things first: a firmware upgrade is not required for SkyConnect at this point.</p>
+    <p>However: We're working on making firmware updates available via Home Assistant.
+      These updates are still under development.
+    </p>
+  </div>
+  <div><h3 class="text-center">SL Web Tools to install firmware updates</h3></div>
+  <p>
+    If you're feeling adventurous and want to try out the latest (experimental) firmware update, you can use
```

Comment:
IMHO, this is a bit too much. Before we had:

>  If you want to make sure that you're running the latest update, you can use the below firmware in the meanwhile. 

There is also another use case: For people which tried the Multiprotocol version. Maybe something along:

"You can use the firmware flasher below to install the latest available firmware versions for Home Assistant SkyConnect. You can also use the firmware updater to flash the Zigbee firmware again if you've previously used the experimental Multiprotocol support."

Reply:
Thank you! Rephrased in new commit. Note: Instead of 'firmware flasher', I used web-flasher, as this is a term that seems to be used with esp home. I'm trying to avoid adding a new term.

Reply:
Sounds good :+1: